### ISSUE TITLE
[IntelligentTransport] TypeLoadException in DSP 0.9.27

### ISSUE THREAD

**Issue by starfi5h** [klarkxy/DysonSphereMod#6]
```
[Info   :   BepInEx] Loading [智能物流计划 1.0.0]
[Info   :   Console] System.TypeLoadException: Could not resolve type with token 0100001c (from typeref, class/assembly AstroPose, Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null)
  at (wrapper managed-to-native) System.Reflection.MonoMethodInfo.get_method_info(intptr,System.Reflection.MonoMethodInfo&)
  at System.Reflection.MonoMethodInfo.GetMethodInfo (System.IntPtr handle) [0x00000] in <2fa7a6a452ca43df998f07fd1486c0df>:0
  at System.Reflection.MonoMethod.GetPseudoCustomAttributes () [0x00002] in <2fa7a6a452ca43df998f07fd1486c0df>:0
  at System.MonoCustomAttrs.GetPseudoCustomAttributes (System.Reflection.ICustomAttributeProvider obj, System.Type attributeType) [0x0000a] in <2fa7a6a452ca43df998f07fd1486c0df>:0
  at System.MonoCustomAttrs.GetCustomAttributesBase (System.Reflection.ICustomAttributeProvider obj, System.Type attributeType, System.Boolean inheritedOnly) [0x0001f] in <2fa7a6a452ca43df998f07fd1486c0df>:0
  at System.MonoCustomAttrs.GetCustomAttributes (System.Reflection.ICustomAttributeProvider obj, System.Type attributeType, System.Boolean inherit) [0x00037] in <2fa7a6a452ca43df998f07fd1486c0df>:0
  at System.MonoCustomAttrs.GetCustomAttributes (System.Reflection.ICustomAttributeProvider obj, System.Boolean inherit) [0x00024] in <2fa7a6a452ca43df998f07fd1486c0df>:0
  at System.Reflection.MonoMethod.GetCustomAttributes (System.Boolean inherit) [0x00000] in <2fa7a6a452ca43df998f07fd1486c0df>:0
  at HarmonyLib.PatchTools+<>c__DisplayClass2_0.<GetPatchMethod>b__0 (System.Reflection.MethodInfo m) [0x00000] in <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0
  at System.Linq.Enumerable.FirstOrDefault[TSource] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] predicate) [0x0002c] in <ea9b8dbcfda24ea39c0a3639d0939b66>:0
  at HarmonyLib.PatchTools.GetPatchMethod (System.Type patchType, System.String attributeName) [0x00018] in <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0
  at HarmonyLib.PatchClassProcessor..ctor (HarmonyLib.Harmony instance, System.Type type, System.Boolean allowUnannotatedType) [0x000a4] in <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0
  at HarmonyLib.Harmony.CreateClassProcessor (System.Type type, System.Boolean allowUnannotatedType) [0x00000] in <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0
  at HarmonyLib.Harmony.PatchAll (System.Type type) [0x00000] in <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0
  at HarmonyLib.Harmony.CreateAndPatchAll (System.Type type, System.String harmonyInstanceId) [0x0001e] in <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0
  at IntelligentTransport.IntelligentTransport.Awake () [0x0000b] in <8932f143c7194ba9a46bfdbc852bebcd>:0
[Info   :    智能物流计划] System.TypeLoadException: Could not resolve type with token 0100001c (from typeref, class/assembly AstroPose, Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null)
  at (wrapper managed-to-native) System.Reflection.MonoMethodInfo.get_method_info(intptr,System.Reflection.MonoMethodInfo&)
  at System.Reflection.MonoMethodInfo.GetMethodInfo (System.IntPtr handle) [0x00000] in <2fa7a6a452ca43df998f07fd1486c0df>:0
  at System.Reflection.MonoMethod.GetPseudoCustomAttributes () [0x00002] in <2fa7a6a452ca43df998f07fd1486c0df>:0
  at System.MonoCustomAttrs.GetPseudoCustomAttributes (System.Reflection.ICustomAttributeProvider obj, System.Type attributeType) [0x0000a] in <2fa7a6a452ca43df998f07fd1486c0df>:0
  at System.MonoCustomAttrs.GetCustomAttributesBase (System.Reflection.ICustomAttributeProvider obj, System.Type attributeType, System.Boolean inheritedOnly) [0x0001f] in <2fa7a6a452ca43df998f07fd1486c0df>:0
  at System.MonoCustomAttrs.GetCustomAttributes (System.Reflection.ICustomAttributeProvider obj, System.Type attributeType, System.Boolean inherit) [0x00037] in <2fa7a6a452ca43df998f07fd1486c0df>:0
  at System.MonoCustomAttrs.GetCustomAttributes (System.Reflection.ICustomAttributeProvider obj, System.Boolean inherit) [0x00024] in <2fa7a6a452ca43df998f07fd1486c0df>:0
  at System.Reflection.MonoMethod.GetCustomAttributes (System.Boolean inherit) [0x00000] in <2fa7a6a452ca43df998f07fd1486c0df>:0
  at HarmonyLib.PatchTools+<>c__DisplayClass2_0.<GetPatchMethod>b__0 (System.Reflection.MethodInfo m) [0x00000] in <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0
  at System.Linq.Enumerable.FirstOrDefault[TSource] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] predicate) [0x0002c] in <ea9b8dbcfda24ea39c0a3639d0939b66>:0
  at HarmonyLib.PatchTools.GetPatchMethod (System.Type patchType, System.String attributeName) [0x00018] in <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0
  at HarmonyLib.PatchClassProcessor..ctor (HarmonyLib.Harmony instance, System.Type type, System.Boolean allowUnannotatedType) [0x000a4] in <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0
  at HarmonyLib.Harmony.CreateClassProcessor (System.Type type, System.Boolean allowUnannotatedType) [0x00000] in <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0
  at HarmonyLib.Harmony.PatchAll (System.Type type) [0x00000] in <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0
  at HarmonyLib.Harmony.CreateAndPatchAll (System.Type type, System.String harmonyInstanceId) [0x0001e] in <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0
  at IntelligentTransport.IntelligentTransport.Awake () [0x0000b] in <8932f143c7194ba9a46bfdbc852bebcd>:
```
Can't load the plugin in the latest game (0.9.27.14659)

**Comment by klarkxy** [comment]
I cannot reproduce it, and I can use the mod at latest game.

**Comment by starfi5h** [comment]
The error occurred in mod version 1.0.0 which is now fixed in 1.0.2.
